### PR TITLE
ID edits

### PR DESCRIPTION
**Pull request by sishida**

### CODE REVIEW

File: codewindtechpreview.md [hunk 1]
```diff
@@ -45,14 +45,13 @@ Projects must be located in `<user>/microclimate-workspace`.
...
-4. Enter the `./codewind install` command in the Terminal.
-5. Run the `./codewind` command to see the available commands.
+2. Make the `codewind` file usable with the `chmod 775 codewind` command.
+3. Enter the `./codewind install` command in the Terminal.
+4. Run the `./codewind` command to see the available commands.
 
 #### Installing the VS Code extension
 1. Install VS Code Version 1.27 or later.
-2. Install the [Codewind VS Code extension](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) from the VS Code Marketplace or by searching for `Codewind` in the VS Code Extensions view.
+2. Install the [Codewind VS Code extension](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) from the VS Code Marketplace or by searching for `Codewind` in the VS Code Extensions view. You can also use the [Marketplace Drag to Install button](https://marketplace.eclipse.org/content/codewind#group-external-install-button).
```
Comment (jopit) [inline codewindtechpreview.md]: The `Marketplace Drag` info should go under the Installing Eclipse section, not under Installing VS Code.

Comment (sishida) [reply codewindtechpreview.md]: Sorry, I'll fix that. Thanks for the catch!

File: codewindtechpreview.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Projects must be located in `<user>/microclimate-workspace`.
 
 #### Downloading the release binary on MacOS
 1. Download <a href="download/codewind" class="download-link trackdownload">codewind</a> to a folder on your system.
-2. Make the `codewind` file usable with the `chmod 775 codewind` command.
+2. Make the `codewind` file usable with the `chmod +r codewind` command.
```
Comment (jopit) [inline codewindtechpreview.md]: `chmod -r` removes all permissions.  I think you want `chmod +x`

Comment (sishida) [reply codewindtechpreview.md]: I will update accordingly.

File: codewindtechpreview.md [hunk 3]
```diff
@@ -58,13 +57,12 @@ Projects must be located in `<user>/microclimate-workspace`.
...
-2. Open the [Eclipse IDE](https://marketplace.eclipse.org/content/codewind) and go to **Help**>**Eclipse Marketplace**.
-3. Search for `Codewind`.
-4. Click **Install**.
-5. Finish the wizard and accept licenses as prompted.
-6. When the installation is complete, restart Eclipse.
-7. Open the Codewind Explorer view. Go to **Window**>**Show View**>**Other** and type `Codewind` in the filter field.
-8. Then, select **Codewind Explorer** and click **Open**.
+2. Obtain the plug-in in one of the following ways: 
+    - Open the [Eclipse IDE](https://marketplace.eclipse.org/content/codewind) and go to **Help**>**Eclipse Marketplace**. Then, search for `Codewind`. Click **Install** and finish the wizard and accept licenses as prompted.
+   - You can also use the [Marketplace Drag to Install button](https://marketplace.eclipse.org/content/codewind#group-external-install-button).
```
Comment (jopit) [inline codewindtechpreview.md]: We want to use the drag-to-install to add an install button to the page that the user can drag into their running eclipse to install.  I believe the markdown to add the button looks like this
```
[![Drag to your running Eclipse* workspace. *Requires Eclipse Marketplace Client](https://marketplace.eclipse.org/sites/all/themes/solstice/public/images/marketplace/btn-install.png)](http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=4638524 "Drag to your running Eclipse* workspace. *Requires Eclipse Marketplace Client")
```

**Comment by bsbyrd1** [comment]
Looks good, Sarah.

**Pull request merged**